src/Nethermind/Nethermind.Runner/configs/chiado_archive.cfg

```diff
@@ -2,9 +2,13 @@
   "Init": {
     "MemoryHint": 768000000,
     "ChainSpecPath": "chainspec/chiado.json",
-	"BaseDbPath": "nethermind_db/chiado_archive",
+    "BaseDbPath": "nethermind_db/chiado_archive",
     "LogFileName": "chiado_archive.logs.txt"
   },
+  "TxPool": {
+    "Size": 1024,
+    "BlobsSupport": "StorageWithReorgs"
```

Comment:
Should this become default? Maybe on mainnet release? @marcindsobczak

Reply:
I think that is the plan